concepts/observability-strategy/configurable-tracing/pocs/pipeline-investigations/README.md

```diff
@@ -0,0 +1,469 @@
+| :--: |
+| Fig. 7 Send exports |
+
+| ![Queue Size](assets/queue_size_cls.png) |
+| :--: |
+| Fig. 8 Queue size |
+
+| ![CPU Utilization](assets/cpu_cls.png) |
+| :--: |
+| Fig. 9 CPU Utilization |
+
+| ![Memory Utilization](assets/memory_cls.png) |
+| :--: |
+| Fig. 10 Memory Utilization |
+
+The CLS instance became approximately after 4 minutes unavailable, which can be observed from `Fig. 8`. After 4 minutes of load test, the queue size starts to grow because spans can't be successfully exported.
+In the `Fig. 6` we can observe `~375000` spans without any problem being ingested and exported to the CLS instance, which results in a peak value of `~1550 spans/sec`. From this time, spans will be queued and dropped after few unsuccessfully retries.
+
+```bash
+2022-11-28T17:36:27.969Z info exporterhelper/queued_retry.go:427 Exporting failed. Will retry the request after interval. {"kind": "exporter", "data_type": "traces", "name": "otlp", "error": "rpc error: code = Unavailable desc = unexpected HTTP status code received from server: 503 (Service Unavailable); transport: received unexpected content-type \"text/html\"", "interval": "8.412634113s"}
+2022-11-28T17:36:33.657Z error exporterhelper/queued_retry.go:176 Exporting failed. No more retries left. Dropping data. {"kind": "exporter", "data_type": "traces", "name": "otlp", "error": "max elapsed time expired rpc error: code = Unavailable desc = unexpected HTTP status code received from server: 503 (Service Unavailable); transport: received unexpected content-type \"text/html\"", "dropped_items": 512}
+go.opentelemetry.io/collector/exporter/exporterhelper.(*queuedRetrySender).onTemporaryFailure
+ go.opentelemetry.io/collector@v0.60.0/exporter/exporterhelper/queued_retry.go:176
+go.opentelemetry.io/collector/exporter/exporterhelper.(*retrySender).send
+ go.opentelemetry.io/collector@v0.60.0/exporter/exporterhelper/queued_retry.go:411
+go.opentelemetry.io/collector/exporter/exporterhelper.(*tracesExporterWithObservability).send
+ go.opentelemetry.io/collector@v0.60.0/exporter/exporterhelper/traces.go:134
+go.opentelemetry.io/collector/exporter/exporterhelper.(*queuedRetrySender).start.func1
+ go.opentelemetry.io/collector@v0.60.0/exporter/exporterhelper/queued_retry.go:206
+go.opentelemetry.io/collector/exporter/exporterhelper/internal.(*boundedMemoryQueue).StartConsumers.func1
+ go.opentelemetry.io/collector@v0.60.0/exporter/exporterhelper/internal/bounded_memory_queue.go:61
+```
+
+### Throughput Test
+
+The throughput test ran over one hour to measure average throughput of span ingestion, in the peak times no span should be dropped or refused by the pipeline and overall resource consumption should stay moderate.
+As a backend, Kyma's integrated Jaeger instance is used. To avoid Jaeger getting a bottleneck in the end-to-end pipeline, the Jaeger instance is configured to use `1.5GiB` memory. The in-memory trace storage of Jaeger instance is reduced to the maximum of `500` traces.
```

Comment:
```suggestion
As a backend, Kyma's integrated Jaeger instance is used. To avoid Jaeger becoming a bottleneck in the end-to-end pipeline, the Jaeger instance is configured to use 1.5GiB memory. The in-memory trace storage of Jaeger instance is reduced to the maximum of 500 traces.
```